examples/mapbox-vector-tiles-simple.html

```diff
@@ -0,0 +1,17 @@
+---
+template: example.html
+title: Simple Mapbox vector tiles example
```

Comment:
This is very minor, but by reading this I don't understand off-hand why it's called simple. That would require me to look at the "normal" mapbox-vector-tiles example. I'd call this one simply mapbox-vector-tiles and rename the other one to mapbox-vector-tiles-advanced instead. But as I said, very minor and opinionated.